build.sbt

```diff
@@ -44,7 +45,22 @@ lazy val root = (project in file("."))
   )
   .disablePlugins(AssemblyPlugin, HeaderPlugin)
 
+lazy val `query-language` = (project in file("kafka-connect-query-language"))
+  .settings(
+    settings ++
+      Seq(
+        name := "kafka-connect-query-language",
+        description := "Kafka Connect compatible connectors to move data between Kafka and popular data stores",
```

Comment:
nit-picking the project description might be better if it says it provides a SQL like DSL to configure the connectors behaviour

Reply:
Thanks  - will fix this in my next PR so as to avoid having to wait for the build again.